Dockerfile

```
@@ -1,6 +1,6 @@
-ARG VERSION=unspecified
+ARG VERSION=0.0.1
```

The `VERSION` arg should remain unspecified since it gets passed in at build time.
```suggestion
ARG VERSION=unspecified
```